CRM/Contact/Form/Contact.php

```diff
@@ -1031,7 +1031,7 @@ public function postProcess() {
     elseif (!empty($params['contact_id']) && ($this->_action & CRM_Core_Action::UPDATE)) {
       // figure out which all groups are intended to be removed
       $contactGroupList = CRM_Contact_BAO_GroupContact::getContactGroup($params['contact_id'], 'Added', NULL, FALSE, TRUE, FALSE, TRUE, NULL, TRUE);
-      if (is_array($contactGroupList)) {
+      if (is_array($contactGroupList) && is_array($params['group']) ?? NULL) {
```

Comment:
Putting the `??` operator outside the function won't guard against unset values. Need to put it inside like
```suggestion
      if (is_array($contactGroupList) && is_array($params['group'] ?? NULL)) {
```